docs/DeveloperGuide.adoc

```diff
@@ -1,4 +1,4 @@
-= AddressBook Level 3 - Developer Guide
+= Weme - Developer Guide
```

Comment:
- Figure 4: multiplicity should either be included for all classes or not included at all.
- Figure 4: dotted lines should be long enough that it is visibly "dotted"
- Figure 7: consider the aggregation association for between budget list and budget
- Figure 14 : all tags checked condition should be beside the diamond (where it forks out)

== Architecture ==
- Architecture Diagram is decently intuitive
- Architecture does not have unnecessary, lower level details.
- Description is sufficiently high level, easy to understand and no indiscriminate use of double headed arrows

== Diagrams ==
- UML  diagrams have mostly correct notation
- UML diagrams are relatively simple with the right amount of details.
- Various different types of diagrams have been used, including class, object, sequence and activity diagrams.
- Diagrams are put  into supplement textual description where helpful
- Most diagrams have significant usage and are not repetitive.

== Overall == 
Documentation is easy to follow, format is consistent, professional and well formatted.